### DISCUSSION TITLE
GetComponentsInChildren의 includeInactive가 작동하나요?

### DISCUSSION THREAD

**Discussion by koldshine** [naverz/zepeto-studio-kor#933] [5. Q&A]
비활성화 되어있는 자식까지 포함해서 찾을려고하는데요
includeInactive를 true로 넣어도 안 찾아지네요.
혹시 저 함수가 작동하는지 궁금합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/discussions/463 이슈와 동일한 문제입니다.
월드 1.5.0 버전에서 수정될 예정입니다.
예상 적용 시기는 7월 초 입니다.
감사합니다.

**Reply by koldshine** [reply]
답변 감사합니다
